addons/point_of_sale/models/pos_order.py

```diff
@@ -48,6 +48,8 @@ def _order_fields(self, ui_order):
             'amount_total':  ui_order['amount_total'],
             'amount_tax':  ui_order['amount_tax'],
             'amount_return':  ui_order['amount_return'],
+            'to_invoice': ui_order['to_invoice'] if "to_invoice" in ui_order else False,
```

Comment:
there is no to_invoice or temporary field on the pos_order model, so it won't do anything except a warning in the logs